src/Dapr.Actors/JsonReaderExtensions.cs

```diff
@@ -394,25 +394,32 @@ public static byte ReadValueAsByte(this JsonReader reader)
             {
                 case JsonToken.String:
                     var valueString = (string)reader.Value;
+                    var spanOfValue = valueString.AsSpan();
```

Comment:
@gmanvel Thanks for making the changes, could you please add unit test around this change as well.

Reply:
Yes, the internals are exposed to Test project 
https://github.com/dapr/dotnet-sdk/blob/master/src/Dapr.Actors/properties/AssemblyInfo.cs#L6

Reply:
@amanbha added a unit test